tests/e2e/e2e_controller_restart_test.go

```diff
@@ -22,106 +22,104 @@ var _ = OSMDescribe("Test HTTP traffic from 1 pod client -> 1 pod server before
 	})
 
 func testHTTPTrafficWithControllerRestart() {
-	{
```

Comment:
Strange that we had this scope to begin with. Was this at some point scoping variables... 🤔

Reply:
It has mostly been due this being done in error in one of the initial tests, with other tests being a copy paste of such test files.